source/accessibility.html.erb

```diff
@@ -84,14 +84,58 @@ title: Accessibility – GOV.UK Verify
         <p class="govuk-body">
           GDS is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
         </p>
+        <h2 class="govuk-heading-l">
+          Compliance status
+        </h2>
+        <p class="govuk-body">
+          This website is partially compliant with the <a href="https://www.w3.org/TR/WCAG21/" class="govuk-link">Web Content Accessibility Guidelines version 2.1</a> AA standard, due to the non-compliances listed below.
+        </p>
+        <h3 class="govuk-heading-l">
+          Non-accessible content
+        </h3>
         <p class="govuk-body">
-          This website is fully compliant with the <a href="https://www.w3.org/TR/WCAG21/" class="govuk-link">Web Content Accessibility Guidelines version 2.1</a> AA standard.
+        The content listed below is non-accessible for the following reasons.
+        </p>
+        <h4 class="govuk-heading-l">
+          Non-compliance with the accessibility regulations
+        </h4>
+        <p class="govuk-body">
+          The following content on Verify Status Page is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard. The non-compliances are listed below:
```

Comment:
CHANGE
Verify Status Page

TO
GOV.UK Verify status page

REASON
To be consistent with the earlier reference to the status page.